client/webserver/site/src/html/markets.tmpl

```diff
@@ -410,15 +418,17 @@
                 </div>
               </div>{{- /* END USER ORDERS */ -}}
 
+              <a href="/orders" class="flex-center my-2 plainlink">[[[view order history]]]</a>
+
               {{- /* RECENT MATCHES */ -}}
-              <div id=recentMatchesBox  class="d-flex flex-column align-items-stretch px-3 pb-4">
+              <div id=recentMatchesBox  class="d-flex flex-column align-items-stretch px-3 pb-4 mt-2">
                 <div id="userOrdersHeader" class="text-center fs20 sans-light my-1">[[[Recent Matches]]]</div>
```

Comment:
Maybe a `brdrtop` and `my-3` on these two divs, respectively:

```diff
diff --git a/client/webserver/site/src/html/markets.tmpl b/client/webserver/site/src/html/markets.tmpl
index e3d7dc27..04e6c13f 100644
--- a/client/webserver/site/src/html/markets.tmpl
+++ b/client/webserver/site/src/html/markets.tmpl
@@ -421,8 +421,8 @@
               <a href="/orders" class="flex-center my-2 plainlink">[[[view order history]]]</a>
 
               {{- /* RECENT MATCHES */ -}}
-              <div id=recentMatchesBox  class="d-flex flex-column align-items-stretch px-3 pb-4 mt-2">
-                <div id="userOrdersHeader" class="text-center fs20 sans-light my-1">[[[Recent Matches]]]</div>
+              <div id=recentMatchesBox  class="d-flex flex-column align-items-stretch px-3 pb-4 mt-2 brdrtop">
+                <div id="userOrdersHeader" class="text-center fs20 sans-light my-3">[[[Recent Matches]]]</div>
                 <table id="recentMatchesTable" class="ordertable">
                   <thead>
                     <tr>
```

gives:

![image](https://user-images.githubusercontent.com/9373513/195172476-84e980dc-fdb0-4017-bf76-67d40e9ddd69.png)